r/R/filesystem.R

```diff
@@ -448,6 +448,28 @@ s3_bucket <- function(bucket, ...) {
   SubTreeFileSystem$create(fs_and_path$path, fs)
 }
 
+#' Connect to an Google Storage Service (GCS) bucket
```

Comment:
Google Cloud Storage? "Google Storage Service (GCS)" looks weird, the acronym doesn't match.